packages/cli/openapi-parser/src/v3/extensions/getStreaming.ts

```diff
@@ -0,0 +1,43 @@
+import { OpenAPIV3 } from "openapi-types";
+import { FernOpenAPIExtension } from "./fernExtensions";
+import { getExtension } from "./getExtension";
+
+export type Streaming = StreamingEndpoint | StreamingConditionEndpoint | undefined;
+
+export interface StreamingEndpoint {
+    type: "stream";
+}
+
+export interface StreamingConditionEndpoint {
+    type: "streamCondition";
+    streamConditionProperty: string;
+    responseStream: string;
+    response: string;
+}
+
+export interface StreamConfig {
+    ["stream-condition"]: string;
+    ["response-stream"]: string;
+    response: string;
+}
```

Comment:
We should probably just rename these `Stream`, `StreamEndpoint`, and `StreamConditionEndpoint` (or some other simpler alternative).